docs/software/social.md

```diff
@@ -28,3 +24,227 @@
+
+#### Access to the phpBB website
+
+- URL = `http://<your.IP>/phpBB3`  
+  (from DietPi v6.33 on: `http://<your.IP>/phpbb`)
+- Click the "Install" tab
+- Click "Proceed to next step"
+- Click "Start install"
+
+#### MySQL/MariaDB database details
+
+To enter the MySQL/MariaDB database details:
+
+- Database server hostname or data source name (DSN) = `localhost`
+- Database username = `phpbb3` (from DietPi v6.33 on: `phpbb`)
+- Database name = `phpbb3` (from DietPi v6.33 on: `phpbb`)
+- Database password = `dietpi` (respectively your chosen global software password)
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+#### Create your admin account
+
+This account will be used for full access to phpBB forum.
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+#### Email and Server URL settings
+
+Use the default values.
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+The database tables will now be generated, please wait.
+
+#### Login
+
+Click "Login"  
+Using your admin login details that you created previously, you can now manage and create your forum pages.
+
+As the installation is now completed, you will need to remove the installation folder, before the forum can be live. Run the following command:
+
+```sh
+rm -R /var/www/phpBB3/install
+# from DietPi v6.33 on:
+rm -R /var/www/phpbb/install
+```
+
+Your forum is now ready.
+
+## Wordpress - Website Blog and Publishing platform
+
+WordPress is a state-of-the-art semantic personal publishing platform with a focus on aesthetics, web standards, and usability.
+It allows you to create your own website.
+
+![DietPi social software Wordpress](../assets/images/dietpi-software-social-wordpress.jpg){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>/wordpress`
+
+=== "First time connect"
+
+    The first time you connect, enter the following MySQL database details:
+
+    - Database name = `wordpress`
+    - Username = `wordpress` (Previous to DietPi v6.0 = `root`)
+    - Password = your global application password
+    - Database Host = `localhost`
+    - Table Prefix = `wp_`
+
+## Image Gallery - Host and browse your images from a web interface
+
+*Image Gallery* allows you to host and browse your images from a web interface.
+
+Also Installs:
+
+- LASP Webserver
+
+![DietPi social software Image Gallery](../assets/images/dietpi-software-social-imagegallery.png){: style="width:500px"}
+
+=== "Access to Image Gallery"
+
+    URL = `http://<your.IP>/gallery`
+
+=== "Add images"
+
+    To add your own images:
+
+    - Create your image folder
+
+      ```sh
+      mkdir /var/www/gallery/MyImageFolder
+      ```
+
+    - Put an image file inside your new folder
+
+      ```sh
+      wget http://media.moddb.com/cache/images/articles/1/153/152601/crop_120x90/download.png
+      mv download.png /var/www/gallery/MyImageFolder/
```

Comment:
Easier:
```
wget http://media.moddb.com/cache/images/articles/1/153/152601/crop_120x90/download.png -O /var/www/gallery/MyImageFolder/download.png
```
At least let's not use a non-HTTPS resource as example (although Tr-Zero is from Dan's Fuzon Design), but our logo instead: https://dietpi.com/images/dietpi-logo_192x192.png
But also much longer.

Reply:
I changed it to:

      ```sh
      wget https://dietpi.com/images/dietpi-logo_192x192.png  
      mv dietpi-logo_192x192.png /var/www/gallery/MyImageFolder/
      ```

I would rather keep the two commands, for most of the readers I assume it's easier for them to understand what happens.